src/adcm_pytest_plugin/steps/common.py

```diff
@@ -38,24 +39,26 @@ def assume_step(title, exception=None):
         raise ValueError("This is expected exception")
 
     """
-    if callable(title):
+    if callable(title):  # pylint: disable=no-else-return
```

Comment:
I can clearly see a return on else clause here. Could you clarify why this mute is here?

Reply:
similar fix https://github.com/arenadata/adcm_pytest_plugin/pull/54/files#diff-c7d550aaf609be6873c469b90e640e7da283e1e2db0bf54736f5b92109c689e6R182-R186
it's a strange inspection and I don't really want to fix it here

Reply:
fix